File: control-flow/Q133/main.go
package main

import "fmt"

func main() {
	nums := [4]int{1, 2, 4, 9}
	count := 0
	for _, i := range nums {
		for _, j := range nums {
			for _, c := range nums {
				for _, k := range nums {
					if i == j || i == c || i == k || j == c || j == k || c == k {
						continue
					} else {
						fmt.Println(i*1000 + j*100 + c*10 + k)
						count++
					}
				}
			}
		}
	}
	fmt.Println("Count:", count)
}

File: control-flow/Q133/main.rs
use std::io;

fn main() {
    let _ = io::stdin();

    let nums = [1, 2, 4, 9];
    let mut count = 0;

    for &i in &nums {
        for &j in &nums {
            for &c in &nums {
                for &k in &nums {
                    if i == j || i == c || i == k || j == c || j == k || c == k {
                        continue;
                    } else {
                        println!("{}", i * 1000 + j * 100 + c * 10 + k);
                        count += 1;
                    }
                }
            }
        }
    }

    println!("Count: {}", count);
}